docs/recipes/real_time_proc.qmd

```diff
@@ -0,0 +1,107 @@
+---
+title: "Real-Time Processing"
+execute:
+  eval: false
+---
+
+
+This recipe serves as an example to showcase the real-time processing capability of DASCore. Here, we demonstrate how to use DASCore to perform rolling mean processing on a spool in real time for edge computing purposes.
```

Comment:
I suggest using the term "near real-time batch processing". True real-time usually implies some kind of high frequency streaming.